readme.md

```diff
@@ -12,3 +12,5 @@
 6. Представить данные в виде AST
 7. Осуществить задачу similar dev search
 
+## Оценка:
+Формула оценки: оценка = github_api * 0.15 + git * 0.25 + enry1 * 0.2 + tree_sitter2 * 0.25 + similar_dev_search * 0.15
```

Comment:
please add gitignore like [this](https://github.com/github/gitignore/blob/main/Python.gitignore)